### PR TITLE
Bratseth/distribute each constant once

### PR DESCRIPTION
**Pull request by bratseth**
@lesters please review

I recommend looking at each commit separately in order.

### CODE REVIEW

File: config-model/src/main/java/com/yahoo/searchdefinition/expressiontransforms/ConvertedModel.java [hunk 1]
```diff
@@ -770,4 +766,47 @@ private static boolean isQuoteSign(int c) {
 
     }
 
+    /**
+     * Models used in a rank profile has the rank profile name as name space while gGlobal model names have no namespace
```
Comment (lesters) [inline config-model/src/main/java/com/yahoo/searchdefinition/expressiontransforms/ConvertedModel.java]: gGlobal

File: config-model/src/main/java/com/yahoo/vespa/model/ml/ModelName.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
+package com.yahoo.vespa.model.ml;
```
Comment (lesters) [inline config-model/src/main/java/com/yahoo/vespa/model/ml/ModelName.java]: header

File: config-model/src/main/java/com/yahoo/searchdefinition/RankingConstant.java [hunk 3]
```diff
@@ -49,14 +56,16 @@ public void setUri(String uri) {
...
 
-    /**
-     * Set the internally generated reference to this file used to identify this instance of the file for
-     * file distribution.
-     */
-    public void setFileReference(String fileReference) { this.fileReference = fileReference; }
-
     public void setType(TensorType tensorType) { this.tensorType = tensorType; }
 
+    /** Initiate sending of theis constant to some services over file distribution */
```
Comment (lesters) [inline config-model/src/main/java/com/yahoo/searchdefinition/RankingConstant.java]: theis

**Review by lesters — APPROVED** [review]
LGTM!

**Comment by bratseth** [comment]
Thanks, fixed.

**Pull request merged**